projects/ansible/collections/dev_guidelines/releasing_collections.rst

```diff
@@ -0,0 +1,73 @@
+* Therefore, please make sure that the current major release of your collection included in 3.0.0 receives at least bugfixes as long new 3.X.X releases are produced.
+* Since new minor releases are included, you can include new features, modules and plugins. You must make sure that you do not break backwards compatibility! (See `semantic versioning <https://semver.org/>`_.) This means in particular:
+
+  * You can fix bugs in **patch** releases but not add new features or deprecate things.
+  * You can add new features and deprecate things in **minor** releases, but not remove things or change behavior of existing features.
+  * You can only remove things or make breaking changes in **major** releases.
+* Make sure that if a deprecation is added in a collection version that is included in 3.x.x, the removal itself will only happen in a collection version included in 5.0.0 or later.
+* Make sure that the policy of releasing, versioning and deprecation is announced to contributors and users in some way. For an example of how to do this, see `the announcement in community.general <https://github.com/ansible-collections/community.general/issues/582>`_. You could also do this in the README.
+
+.. _changelogs:
+
+Changelogs
+==========
+
+Collections are required to include a changelog. To give a consistent feel for changelogs across collections and ensure changelogs exist for collections included in the ``ansible`` package we suggest using `antsibull-changelog <https://github.com/ansible-community/antsibull-changelog>`_ to maintain and generate this.
+
+Before releasing, check:
+
+* all merged pull requests since the last release (except ones related to documentation and new modules/plugins), have `changelog fragments <https://docs.ansible.com/ansible/devel/community/development_process.html#creating-a-changelog-fragment>`_
+* all the fragments follow the `changelog entry format <https://docs.ansible.com/ansible/devel/community/development_process.html#changelog-fragment-entry-format>`_
+
+Release guidelines
+==================
+
+There are several approaches how to release.
+
+If you are not aware of which approach should be used, ask in the ``#ansible-community`` IRC channel.
+
+Releasing without release branches
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use releasing without release branches when:
+
+  * there have not been major releases of the collection before
+  * there have not been breaking changes introduced since the ``1.0.0`` release of the collection
```

Comment:
According to semver, these two are basically equivalent :)

Reply:
I'll try to rephrase

Reply:
@felixfontein I recalled why i use 2 sentences: there can be no major releases since 1.0.0 but at the same time breaking changes can be introduced but not released yet. What do you think?

Reply:
Ah, yeah, sounds good :)